.gitignore

```diff
@@ -19,6 +19,14 @@ test/benchmark/baseline.txt
 xcuserdata/
 *.xccheckout
 
+# Allow Visual Studio project files.
+!wren/
+!wren_lib/
```

Comment:
The current gitignore ignores `wren` in build outputs so the first one might be required. There is probably a better solution to this, though.